modules/combined/test/tests/mortar_tm/2d/ad_frictionless_sec/finite.i

```diff
@@ -108,24 +108,28 @@ name = 'finite'
   [./left_x]
     type = DirichletBC
     variable = disp_x
+    preset = false
```

Comment:
Is there guidance about why this needs to be? I found that PresetBC caused issues when applied to a moving boundary. Is this, and other experience where preset is bad, captured in the doco?

Reply:
You're correct in that the moving boundary is the reason here as to why we need `preset = false` here (with the new default `preset = true`). I'm actively working on these doco changes + a few checks (warnings) for cases like this. These changes didn't get in to the original PR as it was already too large and we wanted to get a start on introducing the warnings (obnoxious legacy warning) for the default change.